zulipterminal/config/ui_mappings.py

```diff
@@ -73,3 +73,19 @@
     4: "Only organization administrators and moderators can edit topic",
     5: "Any user can edit topic",
 }
+
+SPECIAL_KEYS_MAPPINGS = {
+    "meta": "Meta",
+    "esc": "Esc",
+    "ctrl": "Ctrl",
+    "page up": "PgUp",
+    "page down": "PgDn",
+    "end": "End",
+    "enter": "Enter",
+    "tab": "Tab",
+    "shift": "Shift",
+    "up": "\u2191",
+    "down": "\u2193",
+    "left": "\u2190",
+    "right": "\u2192",
```

Comment:
Yes, these 4 characters represent those symbols in symbols.py. So, what is the preferred way of rendering these symbols? From symbols.py or directly by their unicode?

Reply:
As per your other comment, I will remove these symbols from this PR for now.

Reply:
Right, I answered about removal there.

> The idea is that if we add something which is part of the UI or represents a choice of what to display, that we define it centrally and then refer to it elsewhere.

By centrally I meant symbols.py. `up` can then refer to that symbol. That avoids duplication upon adding initial features, as well as accidental duplication later - this is like avoiding 'magic numbers' in code, if you've heard that approach.

While the unicode number is clearer in some respects if one doesn't have the character in a font, we've used the codepoint itself in symbols.py since we're targeting one middle-ground 'symbol-theme' for now.